components/clientComponents/forms/Review/Review.tsx

```diff
@@ -53,6 +57,15 @@ function formatElementValues(element: ReviewElement) {
     );
   }
 
+  // Case of Address Complete
+  if (element.element?.type === FormElementTypes.addressComplete) {
+    if (element.element.properties?.full === false) {
+      return element.values as string; // We're a split address, broken into components.
+    } else {
+      return getAddressAsString(JSON.parse(element.values as string) as AddressElements);
```

Comment:
You may want to consider unlikely the case of invalid JSON. You could use the `safeJSONParse()`  util function. 
For example:
`getAddressAsString(safeJSONParse(element.values as string) as AddressElements)`

All it does is return undefined if there is a thrown JSON parse exception and leave it up to the caller to handle it. We(I) added this util recently and probably forgot to mention it :)

Oh and I see the formattedDate block above (unrelated to this PR) doesn't handle invalid JSON either. Maybe this could be a followup PR instead since there probably other cases.

Reply:
Yah, there's a set of error handling I want to add. :)

Added #4341